### PR TITLE
Discovery ruby names update

### PR DESCRIPTION
**Pull request by tcoffee-google**

### CODE REVIEW

**Comment by tcoffee-google** [comment]
We're still getting an error in datastore that appears not due to discovery updates. Investigating ...

File: src/main/resources/com/google/api/codegen/ruby/apiary_names.yaml.old [hunk 1]
```diff
@@ -2765,6 +2765,7 @@
 "/cloudresourcemanager:v1beta1/key": key
 "/cloudresourcemanager:v1beta1/quotaUser": quota_user
 "/cloudresourcemanager:v1beta1/cloudresourcemanager.projects.create": create_project
+"/cloudresourcemanager:v1beta1/cloudresourcemanager.projects.create/useLegacyStack": use_legacy_stack
```
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/ruby/apiary_names.yaml.old]: Why is this file have an `.old` suffix?

File: src/main/resources/com/google/api/codegen/ruby/apiary_names_update.sh [hunk 2]
```diff
@@ -1,17 +0,0 @@
-#!/usr/bin/env bash
```
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/ruby/apiary_names_update.sh]: Has this script moved somewhere? Is the process info copied somewhere?

**Comment by tcoffee-google** [comment]
I believe datastore discovery got fixed. Our generator should now be consistent with Ruby client library pending https://github.com/google/google-api-ruby-client/pull/478.

File: src/main/resources/com/google/api/codegen/ruby/apiary_names_update.sh (hunk 2 above)
Comment (tcoffee-google) [inline src/main/resources/com/google/api/codegen/ruby/apiary_names_update.sh]: This script is no longer relevant; it was previously used to generate a Ruby names file from our baseline discovery docs to allow our generator tests to pass. (The process logic came directly from the generate script in the client library.) But we've now copied over the full names file from the updated client library, which covers all the APIs.

File: src/main/resources/com/google/api/codegen/ruby/apiary_names.yaml.old (hunk 1 above)
Comment (tcoffee-google) [inline src/main/resources/com/google/api/codegen/ruby/apiary_names.yaml.old]: This was a backup made by the apiary_names_update.sh script for comparison purposes. I'll remove it.

**Comment by tcoffee-google** [comment]
@vchudnov-g PTAL

File: src/main/java/com/google/api/codegen/DiscoveryImporter.java [hunk 3]
```diff
@@ -41,7 +41,7 @@
  * DiscoveryImporter parses a Discovery Doc into a {@link com.google.api.Service} object and an
  * additional {@link ApiaryConfig} object for data not accommodated by Service.
  *
- * The current implementation also provides a main method used for manual sanity checking.
+ * <p>The current implementation also provides a main method used for manual sanity checking.
```
Comment (vchudnov-g) [inline src/main/java/com/google/api/codegen/DiscoveryImporter.java]: We're getting these <p> tags again....

File: src/main/resources/com/google/api/codegen/ruby/apiary_names.yaml.old (hunk 1 above)
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/ruby/apiary_names.yaml.old]: OK

File: src/main/resources/com/google/api/codegen/ruby/apiary_names_update.sh (hunk 2 above)
Comment (vchudnov-g) [inline src/main/resources/com/google/api/codegen/ruby/apiary_names_update.sh]: OK

File: src/main/java/com/google/api/codegen/DiscoveryImporter.java (hunk 3 above)
Comment (vchudnov-g) [inline src/main/java/com/google/api/codegen/DiscoveryImporter.java]: Your editor settings, maybe?

**Comment by vchudnov-g** [comment]
LGTM

File: src/main/java/com/google/api/codegen/DiscoveryImporter.java (hunk 3 above)
Comment (tcoffee-google) [inline src/main/java/com/google/api/codegen/DiscoveryImporter.java]: I think it happens when I run google-java-format. It suspect it applies changes that are not mandated by the verifier.

**Pull request merged**